### ISSUE TITLE
standings: series 'none' does not exist exception

### ISSUE THREAD

**Issue by louking** [louking/rrwebapp#173]
Was able to reproduce this by doing following
- go to standings page
- click settings gear
- click cancel
  
  Message type:       ERROR
        Location:           /home/content/89/11476389/html/scoretility/rrwebapp/standings.py:188
        Module:             standings
        Function:           get
        Time:               2015-11-27 04:18:05,019
  
  ```
    Message:
  
    Error: series 'None' does not exist for 'Frederick Steeplechasers Running Club' club
  ```

**Comment by louking** [comment]
Should fix this by uploading full tree (clubs > years > series) to browser rather than updating select via api

**Comment by louking** [comment]
closed in favor of #360
